lib/collapsers/binary.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+var logger = require('bole')('collapsify:collapsers:binary');
+var base64Utils = require('../utils/base-64');
+
+function external(url, opts) {
+  logger.info('Fetching external binary from "%s".', url);
+
+  return opts.fetch(url).then(collapse);
```

Comment:
I just mean that you can export only `collapse` from each plugin, and when you need to resolve smth, use top-level collapsify.